pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>gov.cms.madie</groupId>
   <artifactId>madie-java-models</artifactId>
-  <version>0.4.0-SNAPSHOT</version>
+  <version>0.3.9-SNAPSHOT</version>
```

Comment:
any reason we are downgrading?

Reply:
@adongare Downgrading because Ethan did upgrading to use jakarta validations, but it breaks the validation as measure-service is still using spring boot 2.6.14, so for now we are downgrading to javax.